home-assistant-plugin/custom_components/xknx/__init__.py

```diff
@@ -68,6 +69,7 @@
         vol.Inclusive(CONF_XKNX_FIRE_EVENT_FILTER, 'fire_ev'):
             vol.All(cv.ensure_list, [cv.string]),
         vol.Optional(CONF_XKNX_STATE_UPDATER, default=True): cv.boolean,
+        vol.Optional(CONF_XKNX_RATE_LIMIT, default=0.05): cv.float,
```

Comment:
```suggestion
        vol.Optional(CONF_XKNX_RATE_LIMIT, default=20):
            vol.All(vol.Coerce(int), vol.Range(min=1, max=100)),
```
I think 100 would be about the maximum telegram rate a KNX TP installation could handle.